bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/pencil-draw-listener/component.jsx

```diff
@@ -208,6 +213,12 @@ export default class PencilDrawListener extends Component {
     window.removeEventListener('touchcancel', this.handleTouchCancel, true);
   }
 
+  contextMenu(event) {
+    this.funcName = 'contextMenu';
```

Comment:
where are you using this property?

Reply:
It is nothing. I added it because lint showed me an error about expecting using “this” inside the function

Reply:
![capture](https://user-images.githubusercontent.com/26368353/42591656-4caa9fa8-8515-11e8-8f9b-79a08ea88c15.PNG)

Reply:
did you tried to see the docs for this linter error? https://eslint.org/docs/rules/class-methods-use-this

Reply:
Hmm... how about adding return this at the end line of the function?

Reply:
I tried to make the function static but it showed me an error